dtfs-central-api/src/helpers/index.ts

```diff
@@ -5,7 +5,9 @@ export { calculateTotalCurrencyAndAmount } from './calculate-total-currency-and-
 export { feeRecordsAndPaymentsMatch } from './fee-record-matching';
 export {
   FeeRecordPaymentEntityGroup,
-  getFeeRecordPaymentEntityGroupsFromFeeRecordEntities,
-} from './get-fee-record-payment-entity-groups-from-fee-record-entities';
+  getFeeRecordPaymentEntityGroups,
+  getFeeRecordPaymentEntityGroupStatus,
+  getFeeRecordPaymentEntityGroupReconciliationData,
+} from './fee-record-payment-entity-group.helper';
```

Comment:
Some of our helpers used this convention, but going to remove it. However, some files also use camel case instead of kebab case, so there is a lack of consistency in general in this directory which should be addressed by DTFS